doc build issue with sphinx_nbexamples and nbconvert

It feels awkward posting this technical issue here, where we are usually discussing general pangeo stuff, but this is also the doc repo, so...

I just pushed some minor changes to the docs and found that the latest build failed:
https://travis-ci.org/pangeo-data/pangeo/builds/525221294

The error seems related to sphinx_nbexamples (0.3.1) and nbconvert (5.5.0). I assume somebody made a release of something and we didn't have a proper version pinned.

```
$ cd docs && make html && cd ..
Running Sphinx v1.6.5
making output directory...
Adding copy buttons to code blocks...
loading pickled environment... not yet created
loading intersphinx inventory from http://xarray.pydata.org/en/latest/objects.inv...
/home/travis/build/pangeo-data/pangeo/docs/conf.py:228: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  people = yaml.load(people_data_file)
/home/travis/build/pangeo-data/pangeo/docs/conf.py:232: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  deployments = yaml.load(deployments_data_file)
Exception occurred:
  File "/home/travis/miniconda/envs/docs-environment/lib/python3.6/site-packages/sphinx_nbexamples/__init__.py", line 383, in create_rst
    raw_rst, resources = nbconvert.export_by_name('rst', nb)
AttributeError: module 'nbconvert' has no attribute 'export_by_name'
The full traceback has been saved in /tmp/sphinx-err-84_58f6c.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
make: *** [html] Error 1
```

Until we can resolve this, we can't update the main pangeo website.

--- a/docs/requirements.txt
+++ b/docs/requirements.txt
@@ -4,7 +4,7 @@ sphinx_bootstrap_theme
 sphinx_pangeo_theme
 jupyter
 ipython
-nbconvert
+nbconvert==5.4.1
 nbsphinx
 doctr>=1.6.3
 sphinx-nbexamples

